server/logic/sendGradesToLadok.js

```
@@ -56,6 +57,13 @@ const modifiedResults = (getAU, getB) => (result) => {
 }
 
 module.exports = async function sendGradesToLadok (courseId, userSelection, token) {
+  let allowed = await isAllowed(token, courseId)
+
+  if (!allowed) {
+    console.log('The user is not allowed to send grades to Ladok')
+    return []
```

This will look exactly the same as if there were no results to send?  Maybe raise an exception instead?